docs/content/latest/api/ysql/commands/ddl_create_table.md

```diff
@@ -116,12 +116,52 @@ postgres=# CREATE TABLE cars(id int PRIMARY KEY,
 
 ### Table with foreign key constraint
 
+Define two tables with a foreign keys constraint.
 ```sql
 postgres=# CREATE TABLE products(id int PRIMARY KEY,
                                  descr text);
 postgres=# CREATE TABLE orders(id int PRIMARY KEY,
                                pid int REFERENCES products(id) ON DELETE CASCADE,
                                amount int);
+
```

Comment:
Can you also update the grammar and diagram files to include `references_clause`?